### DISCUSSION TITLE
The 2 most compelling features for me are the abstraction of regions and the super tight iteration loops

### DISCUSSION THREAD

**Discussion by pbiggar** [darklang/dark#3798] [Feedback]
@mikerochip says:
>I love what Dark is doing for reducing cognitive load and undifferentiated heavy lifting on creating backends. The 2 most compelling features for me are the abstraction of regions and the super tight iteration loops. I think it's going to be tough to get wide adoption without more use case coverage and ergonomic integrations though. It would be nice to have answers to questions like "how do I deploy a backend in China?" "How do I create a secure, passwordless mobile app backend?" "What are the best practices for interop with other runtimes?" And esp for game devs like myself: "how can I integrate a dark backend with a game engine frontend?" "How do I (optionally) segregate users by region?" "Will dark be able to support a real-time game?" Etc

**Comment by mikerochip** [comment]
It me
